src/Core/PaginationData.cs

```diff
@@ -1,70 +1,65 @@
 ﻿using System;
+using System.Linq;
 
-namespace Nudes.Paginator.Core
+namespace Nudes.Paginator.Core;
+
+/// <summary>
+/// Contains all pagination data like page number and size
+/// </summary>
+public class PaginationData
 {
     /// <summary>
-    /// Contains all pagination data like page number and size
+    /// Current Page number, page starts at 1
     /// </summary>
-    public class PaginationData
-    {
-        /// <summary>
-        /// Current Page number, page starts at 1
-        /// </summary>
-        public int Page { get; set; }
-
-        /// <summary>
-        /// Number of items per page
-        /// </summary>
-        public int PageSize { get; set; }
+    public int Page { get; set; }
 
-        /// <summary>
-        /// The Sorting field
-        /// </summary>
-        public string Field { get; set; }
-
-        /// <summary>
-        /// The Sorting direction
-        /// </summary>
-        public SortDirection? SortDirection { get; set; }
-
-        /// <summary>
-        /// Total number of items
-        /// </summary>
-        public long Total { get; set; }
+    /// <summary>
+    /// Number of items per page
+    /// </summary>
+    public int PageSize { get; set; }
 
-        /// <summary>
-        /// Total number of pages
-        /// </summary>
-        public int PageCount => PageSize == 0 ? 0 : (int)Math.Ceiling(Total / (double)PageSize);
+    /// <summary>
+    /// Sequence of sorting definitions in which data has been sorted into
+    /// </summary>
+    public SortingDefinition[] Sorting { get; set; }
```

Comment:
Should be IEnumerable?

Reply:
Receive generic types and return concrete types
In this case we are making sure this is being converted to an array on our side which means a better performance overall, and for consumers of this API arrays implement IEnumerable<T>